src/index.html

```diff
@@ -10,23 +10,43 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search bar</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="/src/styles/style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="big"
+      class="search-form"
+      action="#"
+      method="get"
+    >
+      <div class="search-form__content search-form__content--big">
+        <span class="search-logo search-logo--big"></span>
+        <input
+          class="search-form__input search-form__input--big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </div>
+    </form>
+    <form
+      data-qa="small"
+      class="search-form"
+      action="#"
```

Comment:
The `action` attribute is currently set to `#`. This should be replaced with a placeholder or a relevant URL as per the task requirements to ensure the form functions correctly.